### PR TITLE
BAEL-3598: Improvement: Spring Vault @Secret Annotation

### PR DESCRIPTION
**Pull request by azhwani**
Simple PR

### CODE REVIEW

File: spring-vault/pom.xml [hunk 1]
```diff
@@ -38,7 +54,7 @@
     </dependencies>
 
     <properties>
-        <spring.vault.core.version>2.1.1.RELEASE</spring.vault.core.version>
+        <spring.vault.core.version>2.2.0.RELEASE</spring.vault.core.version>
```
Comment (JavierPeris) [inline spring-vault/pom.xml:57]: Could we try with version 2.2.3.RELEASE ?

File: spring-vault/src/test/java/com/baeldung/springvault/VaultIntegrationTest.java [hunk 2]
```diff
@@ -32,6 +35,9 @@ public class VaultIntegrationTest {
 
     @Autowired
     private CredentialsService credentialsService;
+
```
Comment (JavierPeris) [inline spring-vault/src/test/java/com/baeldung/springvault/VaultIntegrationTest.java:38]: There´re some trailing whitespaces, please remove them.

File: spring-vault/src/test/java/com/baeldung/springvault/VaultIntegrationTest.java [hunk 3]
```diff
@@ -71,5 +77,42 @@ public void whenAccessCredentials_thenCredentialsRetrieved() throws URISyntaxExc
         assertEquals("username", credentials.getUsername());
         assertEquals("password", credentials.getPassword());
     }
+
```
Comment (JavierPeris) [inline spring-vault/src/test/java/com/baeldung/springvault/VaultIntegrationTest.java:80]: There´re some trailing whitespaces, please remove them.

File: spring-vault/src/main/java/com/baeldung/springvault/CredentialsService.java [hunk 4]
```diff
@@ -18,6 +19,9 @@ public class CredentialsService {
 
     @Autowired
     private VaultTemplate vaultTemplate;
+
```
Comment (JavierPeris) [inline spring-vault/src/main/java/com/baeldung/springvault/CredentialsService.java:22]: There´re some trailing whitespaces, please remove them.

File: spring-vault/src/main/java/com/baeldung/springvault/CredentialsService.java [hunk 5]
```diff
@@ -40,5 +44,15 @@ public Credentials accessCredentials() throws URISyntaxException {
         VaultResponseSupport<Credentials> response = vaultTemplate.read("credentials/myapp", Credentials.class);
         return response.getData();
     }
+
```
Comment (JavierPeris) [inline spring-vault/src/main/java/com/baeldung/springvault/CredentialsService.java:47]: There´re some trailing whitespaces, please remove them.

File: spring-vault/pom.xml (hunk 1 above)
Comment (azhwani) [reply spring-vault/pom.xml]: done

File: spring-vault/src/main/java/com/baeldung/springvault/CredentialsService.java (hunk 5 above)
Comment (azhwani) [reply spring-vault/src/main/java/com/baeldung/springvault/CredentialsService.java]: done

File: spring-vault/src/test/java/com/baeldung/springvault/VaultIntegrationTest.java (hunk 2 above)
Comment (azhwani) [reply spring-vault/src/test/java/com/baeldung/springvault/VaultIntegrationTest.java]: done

File: spring-vault/src/test/java/com/baeldung/springvault/VaultIntegrationTest.java (hunk 3 above)
Comment (azhwani) [reply spring-vault/src/test/java/com/baeldung/springvault/VaultIntegrationTest.java]: done

**Pull request merged**